src/io.ts

```diff
@@ -109,7 +110,17 @@ export function readTodoStorageFile(todoStorageFilePath: string): Operation[] {
  * @param operations - An array of string operations that are used to recreate todos.
  */
 export function writeTodoStorageFile(todoStorageFilePath: string, operations: Operation[]): void {
-  writeFileSync(todoStorageFilePath, operations.join(EOL));
+  writeFileSync(todoStorageFilePath, operations.join(EOL) + EOL);
+}
+
+/**
+ * Appends the operations to the .lint-todo storage file to the path provided by the todoStorageFilePath
+ *
+ * @param todoStorageFilePath - The .lint-todo storage file path.
+ * @param operations - An array of string operations that are used to recreate todos.
+ */
+export function appendTodoStorageFile(todoStorageFilePath: string, operations: Operation[]): void {
+  appendFileSync(todoStorageFilePath, operations.join(EOL) + EOL);
```

Comment:
added an equivalent function to `writeTodoStorageFile` called `appendTodoStorageFile`. Placing it next to the write function means it'll be easier to compare the two and ensure they do the same thing.

Reply:
I believe we used to have an append variant of the storage file writing, but collapsed it down to the single function. I'm fine to have a distinct function for append if that makes the code more clear.